docs/reference/datatiers.asciidoc

```diff
@@ -10,8 +10,8 @@ typically share the same hardware profile:
 and hold your most recent, most-frequently-accessed data.
 * <<warm-tier, Warm tier>> nodes hold time series data that is accessed less-frequently
 and rarely needs to be updated.
-* <<cold-tier, Cold tier>> nodes hold time series data that is accessed infrequently and not normally updated.
-* <<frozen-tier, Frozen tier>> nodes hold time series data that is accessed rarely and never updated, kept in searchable snapshots.
+* <<cold-tier, Cold tier>> nodes hold time series data that is accessed infrequently and not normally updated. <<fully-mounted,Fully mounted indices>> of <<ilm-searchable-snapshot,{search-snaps}>> are kept on cold tier eliminating the need for replicas with approximately 50% storage reduction compares to warm tiers.
+* <<frozen-tier, Frozen tier>> nodes hold time series data that is accessed rarely and never updated. <<partially-mounted,Partially mounted indices>> of <<ilm-searchable-snapshot,{search-snaps}>> are kept on frozen tier extend the storage capacity even further by 20 times compares to warm tiers.
```

Comment:
```suggestion
* <<frozen-tier, Frozen tier>> nodes hold time series data that is accessed rarely and never updated. <<partially-mounted,Partially mounted indices>> of <<ilm-searchable-snapshot,{search-snaps}>> are kept on frozen tier to extend the storage capacity even further by up to 20 times compare to the warm tier.
```

Reply:
One thing I never grasp with English grammar is `tense`.  

In the previous [commit](https://github.com/elastic/elasticsearch/commit/5c97e845db636d3df4dd116c5519b4a2c0512e73#r809549540), you propose to change `compares` to `compared`.  In this sentence, you propose to change `compares` to `compare`.  

What have I missed? :P

Reply:
I had a typo, they should both be `compared`, I fixed my typo now :)